pkg/apis/policy/v1alpha1/upstreamtraffic.go

```diff
@@ -0,0 +1,103 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// UpstreamTraffic defines the policy applicable to traffic destined
+// to an upstream host.
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type UpstreamTraffic struct {
```

Comment:
COmmented on the design, but commenting here for posterity. Traffic would indicate the actual bits flowing through the system, whereas this is Setting for upstream traffic. Therefore I think a more appropriate name would be UpstreamTrafficSetting

Reply:
I chose to keep this name concise similar to other API Kinds, e.g [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/), [Egress](https://release-v1-0.docs.openservicemesh.io/docs/api_reference/policy/v1alpha1/#policy.openservicemesh.io/v1alpha1.Egress) etc. I can see how renaming this to `UpstreamTrafficSetting` would make this less ambiguous, so I'll rename it.